### ISSUE TITLE
HackRF does not appear to be recieving on a specific frequency that is known to be transmitting.

### ISSUE THREAD

**Issue by jeremybox** [greatscottgadgets/hackrf#960]
### Steps to reproduce
1  Connect HackRF to PC
2. Start gqrx (with the hackrf selected/set to default)
3. Click Start DSP processing.
4. (optional) Transmit to a local HAM repeater RX frequency (147.900)
5. Observe signal on the repeater TX  frequency (147.300)

### Expected behaviour
Tell us what you expect should happen
I expect that I would see the local repeater's TX (as well as RX) frequency sending something.

### Actual behaviour
No activity is observed on the repeater TX frequency, but it is observed on the RX frequency.  (This is both during a TX action initiated by myself, and from a remote station.  As a control a handheld radio is tuned to both frequencies and can hear both TX to and RX from the repeater.  I strongly suspect that I have damaged the HackRF by transmitting too close to its antenna.

### Version information
**Operating system**:  
Linux purple 5.11.0-34-generic #36-Ubuntu SMP Thu Aug 26 19:22:09 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
Similar behavior is observed in other operating systems such as pentoo and kali

**hackrf_info output:**
hackrf_info version: unknown
libhackrf version: unknown (0.5)
Found HackRF
Index: 0
Serial number: 0000000000000000453c64c82318888f
Board ID Number: 2 (HackRF One)
Firmware Version: local-d6f9d40 (API:1.04)
Part ID Number: 0xa000cb3c 0x00574748

If you are reporting a problem that involves third party software 
(GNU Radio, Gqrx, etc), please report the version here.
tested with gqrx version 2.14.4
I am happy to test with any recommended/desired version.
![Screenshot from Screencast from 22-09-2021 01:56:11 webm](https://user-images.githubusercontent.com/33465600/134224131-68dfea27-c3b0-4067-828c-a7e23ed009f8.png)

### Output
```
Insert any commandline or build output here
```
GQRX output:

gr-osmosdr 0.2.0.0 (0.2.0) gnuradio 3.8.2.0
built-in source types: file fcd rtl rtl_tcp uhd hackrf bladerf rfspace airspy airspyhf soapy redpitaya freesrp 
gr::log :WARN: file_source0 - file size is not a multiple of item size
Resampling audio 96000 -> 48000
BandPlanFile is /home/jeremy/.config/gqrx/bandplan.csv
BookmarksFile is /home/jeremy/.config/gqrx/bookmarks.csv
[INFO] [UHD] linux; GNU C++ version 10.2.1 20201207; Boost_107400; UHD_3.15.0.0-4build1
gr-osmosdr 0.2.0.0 (0.2.0) gnuradio 3.8.2.0
built-in source types: file fcd rtl rtl_tcp uhd hackrf bladerf rfspace airspy airspyhf soapy redpitaya freesrp 
Using HackRF One with firmware local-d6f9d40

**Comment by jeremybox** [comment]
The screenshot is showing the result when transmitting on 147.900, then waiting.  (There was a audible beep from the repeater sent on 147.300, followed by station identifier in morse code, which was heard on handheld radio sitting on my desk, but not picked up by the HackRF

**Comment by jeremybox** [comment]
Additionally, I did not mention, but the HackRF is equipped with a PortaPack running Mayhem firmware.

**Comment by straithe** [comment]
How far away is the station?

**Comment by jeremybox** [comment]
The transmitting station is located 2.4 miles away.  The transmitting station is a repeater transmitting at 100 watts, at 1800' elevation on top of a mountain.  The signal is strong enough that even in my basement, a handheld baofeng can receive a clear voice transmission with the stock antenna, and I am able to hit the offset frequency and chat from inside of my house.  I have tried the hackrf both indoors and outdoors with no noticeable change in behavior.  The hackRF picks up my transmissions on the offset frequency, and prior to my transmitting right next to the antenna, was behaving as I would expect.

**Comment by mossmann** [comment]
I think you've encountered a dynamic range limitation. For some basics on dynamic range see: https://ham.stackexchange.com/questions/2475/why-is-dynamic-range-relevant-for-an-sdr

You've probably configured the HackRF's gain settings fairly low in order to receive a signal from your handheld radio, but that probably drops the signal from the repeater well below the quantization noise floor. You should be able to turn up the gain settings in order to detect the repeater's transmission, but then the gain would be set much too high to be able to receive a signal from the handheld. (At best you would see clipping when transmitting from the handheld, and at worst you would damage your HackRF!)

I suggest that you explore this by turning off your handheld radio and turning up the HackRF's gain until you can detect the signal from the repeater. You may want to enlist the help of a friend who can transmit a signal to the repeater from a safe distance. I suspect that you may need to turn up the total gain by 30 to 40 dB.

I also notice that you may not have tried offset tuning of the repeater frequency. You gqrx screenshot shows the "Hardware Freq" set to 147.3 MHz which is the repeater frequency. This puts the HackRF's DC spike right on top of the repeaters signal which can severely interfere with weaker signals. See: https://hackrf.readthedocs.io/en/latest/faq.html#what-is-the-big-spike-in-the-center-of-my-received-spectrum

In summary, I suggest trying offset tuning first (set the HackRF's "Hardware Freq" to 147.0 MHz for example) and then try turning up the gain as necessary to detect the repeater's signal (but do so without keying up your handheld in proximity to the HackRF).

**Comment by straithe** [comment]
I'm going to close this as there hasn't been a response in a while, but please re-open this issue or open a new one if you still need assistance.

**Comment by jeremybox** [comment]
I hadn't had a chance to revisit this since I had posted initially, but after you closed the ticket I figured out the problem!  Totally user error - me being a dummy.  I was demodulating in the wrong mode (I needed to be on narrow FM).  Thank you for the suggestions, and for revisiting this.

**Comment by straithe** [comment]
Congratulations on the fix! I'm really happy it worked out.
